src/service_control_client_impl_quota_test.cc

```diff
@@ -283,7 +283,7 @@ TEST_F(ServiceControlClientImplQuotaTest, TestCachedQuotaRefreshGotHTTPError) {
   // Check the cached response is the first cached response - QUOTA_EXHAUSTED.
   EXPECT_EQ(quota_response.allocate_errors_size(), 1);
   EXPECT_EQ(quota_response.allocate_errors()[0].code(),
-            error_quota_response1_.allocate_errors[0].code());
+            error_quota_response1_.allocate_errors()[0].code());
```

Comment:
Weren't this not caught by test in last PR?

Reply:
Well, this repo doesn't seem to have a test pipeline for PRs like that in ESPv2.

Reply:
Suggest you clone the repo and run `bazel test //...` locally.